Messaging.md

```diff
@@ -103,19 +137,19 @@ The following DLC-specific types are used throughout the specification. All type
 
 This type contains information about a contract's outcomes, their corresponding payouts, and the oracles to be used.
 
-#### Version 0 `contract_info`
+#### `single_contract_info`
 
-1. type: 55342 (`contract_info_v0`)
+1. type: 1
```

Comment:
I am confused why we are removing these types

Reply:
Before we used type tags in two cases:
* for types that are embedded in others. In this case I couldn't see any use of the tag since if you know exactly what you want to parse a tag doesn't bring anything (happy to be corrected if I missed something).
* for cases where there are multiple version of a field (this case). I couldn't see any reason to use type numbers that are assigned by LN for wire messages, as this is both less efficient and a waste of them AFAICT (again happy to be corrected if I'm wrong).